temba/externals/models.py

```diff
@@ -1,5 +1,9 @@
 from abc import ABCMeta, abstractproperty
 
+from pathlib import Path
```

Comment:
import Path is not being used, to make the code cleaner, unused imports are removed.

Reply:
Fixed